### PR TITLE
supress commits with no diff

### PR DESCRIPTION
**Pull request by phyzical**
This just handles the issue with called /deploy multiple times for the same commit

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ git config user.name "AAF CI"
 git pull
 git add .
 COMMIT_MESSAGE="Update ${projects} image tag ${ECR_REPOSITORY} to '$tag' for ${environments}"
-git commit -m "$COMMIT_MESSAGE"
+git diff-index --quiet HEAD || git commit -m "$COMMIT_MESSAGE"
```
Comment (matthew-puku) [inline entrypoint.sh:39]: I don't understand, if the diff is empty it shouldn't commit anyway because we don't pass `--allow-empty` to `git commit`

Comment (phyzical) [reply entrypoint.sh:39]: proablem is it returns exit code 1 and marks the run as an error

Comment (phyzical) [reply entrypoint.sh:39]: https://github.com/ausaccessfed/federationmanager/actions/runs/4099815929/jobs/7070068733

Comment (matthew-puku) [reply entrypoint.sh:39]: ah yep understood

**Review by matthew-puku — APPROVED** [review]

**Pull request merged**